var/spack/repos/builtin/packages/snappy/package.py

```diff
@@ -25,10 +25,26 @@
 from spack import *
 
 
-class Snappy(AutotoolsPackage):
+class Snappy(CMakePackage):
     """A fast compressor/decompressor: https://code.google.com/p/snappy"""
 
-    homepage = "https://code.google.com/p/snappy"
-    url      = "https://github.com/google/snappy/releases/download/1.1.3/snappy-1.1.3.tar.gz"
+    homepage = "https://github.com/google/snappy"
+    url      = "https://github.com/google/snappy/archive/1.1.7.tar.gz"
 
-    version('1.1.3', '7358c82f133dc77798e4c2062a749b73')
+    version('1.1.7', 'ee9086291c9ae8deb4dac5e0b85bf54a')
+
+    @run_after('install')
+    def install_pkgconfig(self):
+        mkdirp(self.prefix.lib.pkgconfig)
+
+        with open(join_path(self.prefix.lib.pkgconfig, 'snappy.pc'), 'w') as f:
+            f.write('prefix={0}\n'.format(self.prefix))
+            f.write('exec_prefix=${prefix}\n')
```

Comment:
If you don't use `${prefix}` and `${exec_prefix]` anywhere, don't add them.

Reply:
I kept them because these are more or less standard variables that every pkg-config file has (you can query them using `pkg-config --variable`). They are also used in what will hopefully soon be merged upstream (see https://github.com/google/snappy/pull/55). I can of course still remove them if you prefer.